examples/stateful/helm_values/aws-ebs-csi-driver.yaml

```diff
@@ -0,0 +1,2 @@
+node:
```

Comment:
doing this like this means that the default `values.yaml` file is now wiped out - we should set this via `set_values` instead.